awx/ui_next/src/screens/Host/shared/HostForm.jsx

```diff
@@ -30,7 +30,13 @@ function HostForm({ handleSubmit, handleCancel, host, i18n }) {
         inventory: host.inventory || '',
         variables: host.variables,
       }}
-      onSubmit={handleSubmit}
+      onSubmit={async (values, formik) => {
+        try {
+          await handleSubmit(values);
+        } catch (errors) {
```

Comment:
Sometimes, you'll get `errors` back with a `details` field. `details` isn't actually field. When we get these errors, we should treat them as a non-field error.

![Screenshot from 2020-02-05 17-31-49](https://user-images.githubusercontent.com/9753817/73889331-6ef55c00-483d-11ea-954e-0c1945ee7632.png)

^ The way I reproduced this was by attempting to save a host with a name that's already in use. This one should _probably_ be returned as a field error by the API and could probably be quickly patched. That said, we still _need_ to handle `detail` errors in general as it is possible for them to happen in other places.